src/state/index.ts

```diff
@@ -35,7 +34,6 @@ const persistedReducer = persistReducer(
     farms: farmsReducer,
     pools: poolsReducer,
     predictions: predictionsReducer,
-    profile: profileReducer,
```

Comment:
Please remove `profile` from `PERSISTED_KEYS`

Reply:
Done

Reply:
Looks like we'll have error when deleted key. Need to add `blacklist` or add migration for it.
https://github.com/rt2zz/redux-persist/issues/618

Reply:
Done 👍 , added to blacklist